src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -459,9 +459,12 @@ private class HotReloadState : IProjectHotReloadSessionCallback
             // TODO: Support restarting the session.
             public bool SupportsRestart => false;
 
+            public UnconfiguredProject? Project { get; }
+
             public HotReloadState(ProjectHotReloadSessionManager sessionManager)
             {
                 _sessionManager = sessionManager;
+                Project = _sessionManager._project;
```

Comment:
Only a small thing, but we can avoid storing a reference to the project here, and use delegation in the property instead.

```suggestion
            public UnconfiguredProject? Project => _sessionManager._project;

            public HotReloadState(ProjectHotReloadSessionManager sessionManager)
            {
                _sessionManager = sessionManager;
```